src/dpsvalidationservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/ValidationControllerTest.java

```diff
@@ -64,4 +68,11 @@ public void withInvalidSurcodeShouldReturnErrorResponse() {
         GetValidOpenDFCMCase response = sut.getValidOpenDFCMCase("INVALID", "PEL#");
         Assert.assertEquals(DpsValidationServiceConstants.VALIDOPEN_DFCMCASE_ERR_RESPONSE_CD, response.getResult());
     }
+
+    @Test
+    public void withClientThrowingExceptionShouldReturnErrorResponse() {
+        GetValidOpenDFCMCase response = sut.getValidOpenDFCMCase("1234568", "EXP");
+        Assert.assertEquals(DpsValidationServiceConstants.VALIDOPEN_DFCMCASE_ERR_RESPONSE_CD, response.getResult());
```

Comment:
this line looks weird in the diff for the PR but I'm assuming it's indented correctly

Reply:
Looks good on my editor some github weirdness...

![image](https://user-images.githubusercontent.com/51387119/73014384-e6070b00-3dce-11ea-9a31-9a7c48b00fe4.png)

Reply:
100% always git diff weirdness.